apps/editor/index.d.ts

```diff
@@ -729,6 +740,12 @@ declare namespace toastui {
 
     public addWidget(selection: Range, node: Node, style: string, offset?: number): void;
 
+    public addCommand(type: Command): void;
+
+    public addCommand(type: string, props: CommandPropsOptions): void;
```

Comment:
Why did you declare `addCommand twice` ?

Reply:
The first type is different: the first one takes a single Command, the second one takes two parameters: a string and a CommandPropsOptions

https://github.com/nhn/tui.editor/blob/master/apps/editor/src/js/editor.js#L359-L365

I could declare it 

> public addCommand(type: string | Command, props?: CommandPropsOptions): void;

but I thought declaring the two options separately was more precise.